packages/atlas/src/components/_nft/NftWidget/NftHistory.tsx

```diff
@@ -0,0 +1,178 @@
+  NftHistoryHeader,
+  StyledChevronButton,
+  TextContainer,
+  ValueContainer,
+} from './NftHistory.styles'
+import { Size } from './NftWidget'
+import { OwnerHandle } from './NftWidget.styles'
+
+export const NftHistory: React.FC<{ size: Size; width: number }> = ({ size, width }) => {
+  const [isOpen, toggleIsOpen] = useToggle(true)
+  const { convertToUSD } = useTokenPrice()
+
+  const dummyData: HistoryItemProps[] = [
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Bid made by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Listed by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      copy: 'Auction settled by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Auction won by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Bid made by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Listed by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      copy: 'Auction settled by',
+      memberHandle: 'Madness',
+    },
+    {
+      date: new Date(),
+      memberAvatarUri: 'https://picsum.photos/40/40',
+      size,
+      joyAmount: 12334,
+      dollarValue: convertToUSD(12334),
+      copy: 'Auction won by',
+      memberHandle: 'Madness',
+    },
+  ]
+
+  return (
+    <>
+      <NftHistoryHeader data-open={isOpen} data-size={size} onClick={toggleIsOpen}>
+        <Text variant={size === 'small' ? 'h300' : 'h400'}>History</Text>
+        <StyledChevronButton data-open={isOpen} iconOnly variant="tertiary" icon={<SvgActionChevronB />} />
+      </NftHistoryHeader>
+      {isOpen && (
+        <div style={{ position: 'relative' }}>
+          <FadingBlock data-size={size} width={width} />
+          <HistoryPanel width={width} data-size={size} data-open={isOpen}>
+            {/* TODO: remove dummy data */}
+            {dummyData.map((props: HistoryItemProps, index) => (
+              <HistoryItem key={index} {...props} />
+            ))}
+          </HistoryPanel>
+          <FadingBlock data-size={size} width={width} data-bottom />
```

Comment:
Do we need `data-bottom` if it doesn't provide any value?

Reply:
yes for these styles
![image](https://user-images.githubusercontent.com/10615263/158076690-ddf2368d-3eba-4f73-a871-a2d6417060f3.png)